akka-docs-dev/rst/scala/stream-flows-and-basics.rst

```diff
@@ -117,39 +117,39 @@ specification, which Akka is a founding member of.
 The user of the library does not have to write any explicit back-pressure handling code — it is built in
 and dealt with automatically by all of the provided Akka Streams processing stages. It is possible however to add
 explicit buffer stages with overflow strategies that can influence the behaviour of the stream. This is especially important
-in complex processing graphs which may even sometimes even contain loops (which *must* be treated with very special
+in complex processing graphs which sometimes even may contain loops (which *must* be treated with very special
```

Comment:
Google has results for both of course, but "sometimes even may" is almost always reported as "sometimes even May". Let's ask someone.